techniques/failures/F94.html

```diff
@@ -12,146 +12,129 @@
-               of <a href="../../guidelines/sc/20/resize-text.html">Success Criterion 1.4.4</a>.
+            <p>ビューポート単位の使用法によっては、テキストサイズの調整を妨げない場合もあるが、ビューポート単位がテキストサイズを定義するための主要な方法として使用されている場合、<a href="https://waic.jp/docs/WCAG21/Understanding/resize-text">達成基準 1.4.4</a> の失敗の原因となる可能性がある。
             </p>
             <div class="note">
-               <div role="heading" class="note-title marker" aria-level="3">Note</div>
-               <p>If media queries were used to adjust the size of text or unit of measure at different
-                  screen sizes, it may not be a failure of <a href="https://waic.jp/docs/WCAG21/Understanding/resize-text.html">Resize text</a>. On-page controls provided by the author are also a way of passing the resize text
-                  success criteria.
+               <div role="heading" class="note-title marker" aria-level="3">注記:</div>
+               <p>メディアクエリを使用して、異なる画面サイズでテキストのサイズや単位を調整している場合、<a href="https://waic.jp/docs/WCAG21/Understanding/resize-text.html">テキストのサイズ変更</a>の失敗には該当しないかもしれない。コンテンツ制作者がページ上に提供するコントロールもまた、テキストのサイズ変更の達成基準を満たすためのひとつの手段である。
                </p>
             </div>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-failure-example-1">
-               <h3>Example 1: Failure example 1</h3>
-               <p>The following CSS and HTML snippet uses VW units to size the text.</p><pre><code>/* CSS */
+               <h3>例 1: 失敗例 1</h3>
+               <p>以下のCSSとHTMLのスニペットでは、VWの単位を使ってテキストのサイズを設定している。</p><pre><code>/* CSS */
 .callout {
     font-size:1vw;
 }
 
 &lt;p class="callout"&gt;
-  Text that scales by viewport units&lt;p/&gt;</code></pre><p class="working-example">Example <a href="../../working-examples/failure-viewport-units/">page with an example of text sized in <code>vh</code> units</a>.
+  Text that scales by viewport units&lt;p/&gt;</code></pre><p class="working-example"><a href="https://www.w3.org/WAI/WCAG21/working-examples/failure-viewport-units/"><code>vh</code> 単位でテキストのサイズが設定されているページの例</a>.
                </p>
             </section>
          </section>
          <section id="resources">
-            <h2>Resources</h2>
-            <p>Resources are for information purposes only, no endorsement implied.</p>
+            <h2>参考リソース</h2>
+            <p>この参考リソースは、あくまでも情報提供のみが目的であり、推薦などを意味するものではない。</p>
             <ul>
                					
-               <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/length"><abbr title="Mozilla Development Network">MDN</abbr> CSS units</a> including <code>vh</code> and <code>vw</code> units.
+               <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/length">&lt;length&gt; - CSS: Cascading Style Sheets | <abbr title="Mozilla Development Network">MDN</abbr></a> <code>vh</code> 及び <code>vw</code> の単位についての説明が含まれている。
```

Comment:
```suggestion
               <li><a href="https://developer.mozilla.org/ja/docs/Web/CSS/length">&lt;length&gt; - CSS: Cascading Style Sheets | <abbr title="Mozilla Development Network">MDN</abbr></a> <code>vh</code> 及び <code>vw</code> の単位についての説明が含まれている。
```

＃ わりとどうでもいいですが、MDNはもはや何かの略称ではなく単なるMDNなのでもにょりますね…。

Reply:
> MDNはもはや何かの略称ではなく単なるMDNなのでもにょりますね…。

(笑)